stingray/mission_support/rxte.py

```diff
@@ -308,7 +310,7 @@ def rxte_calibration_func(instrument, epoch):
     raise ValueError(f"Unknown XTE instrument: {instrument}")
 
 
-def rxte_pca_event_file_interpretation(hdulist):
+def rxte_pca_event_file_interpretation(input_data, header=None, hduname=None):
```

Comment:
Just fyi, David Gardenier, a former Master's/PhD student at the API, wrote a pipeline for RXTE that can handle all the PCA data types (https://github.com/davidgardenier/chromos). It's pretty outdated and in Python2, but maybe you could get some ideas on how to handle all types of RXTE observations from it?

Reply:
@matteolucchini1 from what I see, it's a long list of wrappers to the XTE pipeline, assuming raw data. Here, I tried to be independent of the pipeline, and starting from the L2 event data which are now available in the RXTE archive